cosmoz-grouped-list.js

```diff
@@ -594,8 +576,8 @@
 		},
 
 		toggleCollapse(item) {
-			let model = this._getModelFromItem(item),
-				itemState = this._itemsMap.get(item);
+			const model = this._getModelFromItem(item);
+			let itemState = this._itemsMap.get(item);
```

Comment:
Declare/set as close to usage as possible.
In this case we can avoid fetching from _itemsMap if we don't have a model